src/components/layout/Navigation.vue

```diff
@@ -11,9 +11,9 @@
       <img src="../../assets/headerLogo.svg">
     </RouterLink>
 
-    <RouterLink :to="{ name: 'tips' }">
+    <RouterLink :to="{ name: 'feed' }">
       <IconTips />
-      {{ $t('tips') }}
+      {{ $t('Feed') }}
```

Comment:
```suggestion
      Feed
```
Anyway we have to adit this when adding the corresponding translations